docs/source/techniques/ISIS_SingleCrystalDiffraction_Workflow.rst

```diff
@@ -176,7 +176,7 @@ The reduction proceeds as follows:
         sxd.save_peak_table(run, peak_type, integration_type, save_dir, save_format='SHELX')
```

Comment:
While you're here, could you add the line
`remove_non_integrated_peaks(sxd.get_peaks(run, peak_type, integration_type))`
between current lines 175 and 176?
It's recently been brought to my attention that SHELX does not no how to handle peaks with 0 intensity (which happened when peaks are ignored because e.g. they are on the edge of the detector).